force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -161,18 +240,19 @@ public with sharing class FlowControlLoopsCollections {
      * @return The new string with the inputString repeated repeatCount times.
      */
     public static String repeatString(String inputString, Integer repeatCount) {
+        
         // Initialize the result String
+        String result = '';
 
-        // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+        for (Integer i = 0; i < repeatCount; i++) {
+            result += inputString;
+            if (i + 1 < repeatCount) {
```

Comment:
Good job checking for the additional separately instead after you add in inputString.